pkg/workflows/wasm/host/wasm.go

```diff
@@ -5,27 +5,89 @@ import (
 	"errors"
 	"fmt"
 
+	"github.com/go-viper/mapstructure/v2"
 	"github.com/google/uuid"
-
 	"google.golang.org/protobuf/types/known/emptypb"
 
-	"github.com/smartcontractkit/chainlink-common/pkg/workflows/sdk"
-	wasmpb "github.com/smartcontractkit/chainlink-common/pkg/workflows/wasm/pb"
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+	"github.com/smartcontractkit/chainlink-common/pkg/values"
+	legacySdk "github.com/smartcontractkit/chainlink-common/pkg/workflows/sdk"
+	legacywasmpb "github.com/smartcontractkit/chainlink-common/pkg/workflows/wasm/pb"
+	wasmpb "github.com/smartcontractkit/chainlink-common/pkg/workflows/wasm/v2/pb"
 )
 
-func GetWorkflowSpec(ctx context.Context, modCfg *ModuleConfig, binary []byte, config []byte) (*sdk.WorkflowSpec, error) {
+func GetTriggersSpec(ctx context.Context, modCfg *ModuleConfig, binary []byte, config []byte) ([]*wasmpb.TriggerSubscriptionRequest, error) {
+	m, err := NewModule(modCfg, binary, WithDeterminism())
+	if err != nil {
+		return nil, fmt.Errorf("could not instantiate module: %w", err)
+	}
+	m.Start()
+	defer m.Close()
+
+	execResult, err := m.Execute(ctx, &wasmpb.ExecuteRequest{
+		Id:      uuid.New().String(),
+		Config:  config,
+		Request: &wasmpb.ExecuteRequest_Subscribe{Subscribe: &emptypb.Empty{}},
+	})
+
+	if err != nil {
+		return nil, err
+	}
+
+	switch r := execResult.Result.(type) {
+	case *wasmpb.ExecutionResult_Value:
+		v, err := values.FromProto(r.Value)
```

Comment:
I thought that we were no longer using values, but strong proto types?

Reply:
For capability calls, yes. For workflow return values and consensus, I left the values.Value package. We don't know the type the user needs for those cases.

There may be something better we can use, @cedric-cordenier mentioned an alternative, but I didn't understand it.